packages/react-scripts/template/README.md

```diff
@@ -1289,6 +1290,19 @@ Install the Surge CLI if you haven't already by running `npm install -g surge`.
 
 Note that in order to support routers that use HTML5 `pushState` API, you may want to rename the `index.html` in your build folder to `200.html` before deploying to Surge. This [ensures that every URL falls back to that file](https://surge.sh/help/adding-a-200-page-for-client-side-routing).
 
+## Advanced Configuration
+
+You can adjust various development and production settings via environment variables.
+
+Variable | Development | Production | Usage
+:--- | :---: | :---: | :---
+BROWSER | :white_check_mark: | :x: | By default, Create React App will open the default system browser, favoring Chrome on macOS. To override this behavior, the browser may be specified or set to `none` to disable this behavior.
```

Comment:
We should at least link to all valid values here. Not sure where the list is, but [this](https://github.com/sindresorhus/opn#app) is a start.